--- src/settings/_default.settings	2016-08-29 22:21:19.000000000 -0600
+++ _default-1.settings	2016-09-03 00:44:42.774965332 -0600
@@ -239,7 +239,7 @@
     "setting": "send_metrics"
   },
   {
-    "value": true,
+    "value": false,
     "title": "",
     "type": "hidden",
     "category": "Tutorial",
@@ -628,4 +628,4 @@
     "value": "Ctrl+I",
     "type": "text"
   }
-]
\ No hay ningún carácter de nueva línea al final del fichero
+]
